### PR TITLE
Use js books class

### PR DESCRIPTION
**Pull request by Bavon101**
**In this section we were able to do the following:**

- [ ] Create `AwesomeBooks class` that has all the necessary functions to manipulate the books.
- [ ] Customized the site to look like the provided `wireframe`
- [ ] Added `CSS styling`

### CODE REVIEW

File: js/books.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+//   removeBook(id) {
+//     this.books = this.books.filter((a) => a.id !== id);
+//     this.saveBooks();
+//   }
+
+//   getStoredBooks() {
+//     this.books = JSON.parse(localStorage.getItem('books'));
+//   }
+// }
```
Comment (FrederickMih) [inline js/books.js]: - [x] Please let's avoid leaving comments in our codebase. Being interrupted by redundant comments can create distractions and make our code looks messy.

File: js/main.js [hunk 2]
```diff
@@ -1,45 +1,65 @@
...
+
+  addBook(title, author) {
+    const book = {
+      title,
+      author,
+      id: Date.now(),
+    };
+    this.books.unshift(book);
+    this.saveBooks();
+  }
```
Comment (FrederickMih) [inline js/main.js:17]: - [ **OPTIONAL**] This method works well to add a book to the books array but it mutates the original array. The `unshift` does not return a new copy of the `books` array. Mutation may lead to unexpected and hard-to-debug issues. Please let's always consider using methods that do not mutate the original array like concat(), filter(), slice().

**Review by FrederickMih — CHANGES_REQUESTED** [review]
##### Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap: :clap: 

:arrow_forward: Your repo looks professional :heavy_check_mark: 
:arrow_forward: Linter correctly set up :heavy_check_mark: 
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @Bavon101,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

### **Note**

- Please take the suggestion from my colleague TSE on the mutated array (unshift).


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**